docs/using.rst

```diff
@@ -168,6 +169,10 @@ the UI, you can use the plugin to obtain a cert by specifying
 to copy and paste commands into another terminal session, which may
 be on a different computer.
 
+Additionally you can specify scripts to run before and after verification
```

Comment:
I guess either "perform authentication" or "prepare for validation" could be better ways of putting it?